testpmd-container-app/testpmd/Dockerfile

```
@@ -51,5 +59,12 @@ RUN dnf install -y \
 
 # copy testpmd runtime cmdline file
 COPY testpmd-runtime-cmds.txt /root/testpmd-runtime-cmds.txt
+
+# Copy scripts
 COPY scripts /usr/local/bin
+COPY --from=build2 /scripts/webserver /usr/local/bin/
+
+# Update permissions
+RUN chmod +x /usr/local/bin/webserver
```

Avoid copying anything that is not needed in the image that is inside the scripts, e.g. the webserver.go

```suggestion
COPY --chmod=550 --from=build2 /scripts/webserver /usr/local/bin/
```